### ISSUE TITLE
ruby_rbenv 

### ISSUE THREAD

**Issue by tylrd** [TalkingQuickly/rails-server-template#40]
Just started going through the book, thanks for the great info.

I ran into an issue with this repo:

```
cookbook 'rbenv', github: 'fnichol/chef-rbenv' 
```

I get this error:

```
In your Berksfile, you have:

  cookbook 'rbenv'

But that cookbook is actually named 'ruby_rbenv'

This can cause potentially unwanted side-effects in the future.

NOTE: If you do not explicitly set the 'name' attribute in the metadata, the name of the directory will be used instead. This is often a cause of confusion for dependency solving.
```

I changed the name to:

```
cookbook "ruby_rbenv", github: "chef-rbenv/ruby_rbenv"
```

And the `bundle exec berks install` works.  However, that version of the repo requires Chef 12.1+, as it uses some new DSL ("provides"?).

I ended up just using more recent versions of knife-solo and chef, and it seems to work, but I could be breaking other things without realizing it.

**Comment by Freika** [comment]
https://github.com/chef-rbenv/ruby_rbenv/issues/126

**Comment by TalkingQuickly** [comment]
Resolved in v4.0 sample code
